js/components/util/files.ts

```diff
@@ -82,25 +79,17 @@ export const loadLocalFile = async (
  * @param url URL, relative or absolute to JSON file
  * @returns true if successful
  */
-
 export const loadJSON = async (disk2: DiskII, number: DriveNumber, url: string) => {
-    return new Promise((resolve, reject) => {
-        fetch(url).then(function (response: Response) {
-            if (response.ok) {
-                return response.json();
-            } else {
-                throw new Error('Error loading: ' + response.statusText);
-            }
-        }).then(function (data: JSONDisk) {
-            if (includes(DISK_FORMATS, data.type)) {
-                disk2.setDisk(number, data);
-            }
-            initGamepad(data.gamepad);
-            resolve(true);
-        }).catch(function (error) {
-            reject(error.message);
-        });
-    });
+    const response = await fetch(url);
+    if (response.ok) {
+        const data: JSONDisk = await response.json();
+        if (includes(NIBBLE_FORMATS, data.type)) {
+            disk2.setDisk(number, data);
+        }
+        initGamepad(data.gamepad);
+    } else {
+        throw new Error('Error loading: ' + response.statusText);
```

Comment:
For consistency, shouldn't we just use template strings:
```suggestion
        throw new Error(`Error loading: ${response.statusText}`);
```